src/current/v23.1/import-performance-best-practices.md

```diff
@@ -44,60 +107,63 @@ IMPORT INTO customers (id, name)
     );
 ~~~
 
-CockroachDB imports the files that you give it, and does not further split them. For example, if you import one large file for all of your data, CockroachDB will process that file on one node—even if you have more nodes available. However, if you import two files (and your cluster has at least two nodes), each node will process a file in parallel. This is why splitting your data into as many files as you have nodes will dramatically decrease the time it takes to import data.
+CockroachDB imports the files that you give it, and does not further split them. For example, if you import one large file for all of your data, CockroachDB will process that file on one node, even if you have more nodes available. However, if you import three files (and your cluster has at least three nodes), each node will process a file in parallel. This is why splitting your data into at least as many files as you have nodes will dramatically decrease the time it takes to import data.
```

Comment:
Nit: Can we say something like "CockroachDB ingests files as provided and does not automatically split them further." Basically the "give it" felt a little too casual but feel free to disregard if this is the consistent voice